src/core/changelogcontents.cpp

```diff
@@ -111,6 +111,23 @@ void ChangelogContents::request()
     changelog = changelog.replace( regexpAllTitles, QStringLiteral( "\n###\n##\\1\n\n\n" ) );
     changelog = "Up to release **" + versionNumbersOnly + "**" + changelog;
 
+#if defined( Q_OS_ANDROID )
+    const QString platformName = QLatin1String( "android" );
+#elif defined( Q_OS_IOS )
+    const QString platformName = QLatin1String( "ios" );
+#elif defined( Q_OS_WIN )
+    const QString platformName = QLatin1String( "windows" );
+#elif defined( Q_OS_MACOS )
+    const QString platformName = QLatin1String( "macos" );
+#elif defined( Q_OS_LINUX )
+    const QString platformName = QLatin1String( "ios" );
+#else
+    const QString platformName = QString();
+#endif
+    QRegularExpression regexpPlatform( QStringLiteral( "<!--\\s*platformbegin:\\s*((?!%1).)*-->(.|\n)*?<!--\\s*platformend\\s*-->" ).arg( platformName ) );
+    qDebug() << "hit? " << ( changelog.contains( regexpPlatform ) ? "yes" : "no" );
```

Comment:
Do we need the `qDebug`? If yes, a slighly more verbose message will make it easier to read the logs.

Reply:
@suricactus , I'm so filthy! :wink: thanks for spotting this.